cabal.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -eu
```

Comment:
Stack locates its 'helper' scripts in `etc/scripts`. It may help to give the file a fuller name, more indicative of its function - see the other script file names for examples.

It would also be helpful to document the intended use of the script at its start (even though it is short).

Reply:
That said, if I am correct about the commands in the script (see below) is not a single command script overkill?